src/platforms/vue-native/scripts/compiler.js

```diff
@@ -243,8 +265,33 @@ function traverse(ast, nodes = []) {
     nodes.push(ast.sourceCodeLocation)
   }
   if (ast.childNodes) {
-    ast.childNodes.forEach(child => {
+    ast.childNodes.forEach((child) => {
       traverse(child, nodes)
     })
   }
 }
+
+function getLineOfExport(content) {
+  var lineOfExport = 0
+  content.split(newLine).some((line, index) => {
+    if (line.includes('export')) {
```

Comment:
This may also match the string `export`, if it is inside a string and not being used as a keyword. Could we check this in a better way?